app/javascript/app/components/table/table-component.jsx

```diff
@@ -31,17 +32,20 @@ class SimpleTable extends PureComponent {
 
     if (!data.length) return null;
     const hasColumnSelectedOptions = hasColumnSelect && columnsOptions;
+    const getResponsiveWidth = width => {
```

Comment:
Could you spend some time to play with the number of columns selected to obtain the ratio?
Then we can obtain more readable tables in mobile, what are your thoughts?